lib/engine/config/tile.rb

```diff
@@ -52,7 +52,7 @@ module Tile
         # '403' => 'city=revenue:30;town=revenue:30;path=a:0,b:_0;label=B;upgrade=cost:40',
         '437' => 'town=revenue:30;path=a:0,b:_0;path=a:_0,b:2;icon=image:port,blocks_lay:1',
         '438' => 'city=revenue:40;path=a:0,b:_0;path=a:2,b:_0;label=H;upgrade=cost:80',
-        '445' => 'town=revenue:20;path=a:0,b:_0;path=a:_0,b:2;icon=image:18_al/tree,blocks_lay:1',
+        '445' => 'town=revenue:20;path=a:0,b:_0;path=a:_0,b:2;icon=image:18_al/tree,blocks_lay:1;skip_check=town_size',
```

Comment:
i don't think we should have this property on the tile, it should be a part of the ability, i also don't know if it's necessary to have this check

Reply:
The property was removed in later versions of this PR.